examples/multiple-results/multiple-results.go

```diff
@@ -108,13 +108,19 @@ func multiValueWasmFunctions(ctx context.Context, r wazero.Runtime) (api.Module,
 //go:embed testdata/multi_value_imported.wasm
 var multiValueFromImportedHostWasm []byte
 
+// multiValueWasmFunctions return the WebAssembly which imports the Go-defined "get_age" function which
+// returns multiple results. The source is in testdata/multi_value_imported.wat
 func multiValueFromImportedHostWasmFunctions(ctx context.Context, r wazero.Runtime) (api.Module, error) {
-	r.NewModuleBuilder("multi-value/host").
+	// Instantiate the host module with the exported `get_age` function which returns multiple results.
+	if _, err := r.NewModuleBuilder("multi-value/host").
```

Comment:
this error hasn't been check, but no idea why our linter didn't catch

Reply:
not sure.. wonder if because it is in the examples dir?